src/content/admin/okteto-insights.mdx

```diff
@@ -1,12 +1,19 @@
 ---
 title: Okteto Insights
-description: Use Okteto Insights for detailed analytics on resource usage, build, and deployment metrics
+description: Configure Okteto Insights for detailed analytics on resource usage, build, and deployment metrics
 sidebar_label: Okteto Insights
 id: okteto-insights
 ---
 
 import Image from "@theme/Image";
 
+This page explains how to configure Okteto Insights for Self-Hosted instances and outlines the data available from our Prometheus endpoints. 
```

Comment:
As mentioned [here](https://github.com/okteto/docs/pull/858/files#diff-ee5c15f1c8dbfc7b8f0cd89a2debeb2a46c7a4d09d2d9be77f88de23529d3908R95), this can also be done in SaaS/BYOC. SaaS/BYOC already have built-in insights dashboards, but they can also build their own scraping the metrics endpoint